app/assets/stylesheets/home.sass

```diff
@@ -1,9 +1,91 @@
 @import variables
 @import home/schedule
 
+.behind
+  position: absolute
+  margin-top: 125px
+  width: 1000px
+  height: 100%
+  z-index: -1
+  top: 0
+  left: 50%
+  transform: translateX(-50%)
+  overflow: visible
+
+  *
+    width: 100%
+    position: absolute
+    background-size: contain
+    background-repeat: no-repeat, no-repeat
+
+  .bg-img1
+    height: 402px
+    width: 440px
+    background-image: url("home_asset_1.png")
```

Comment:
Use `asset-url()` instead - otherwise this will break in production. Same for the other instances below